### DISCUSSION TITLE
Implement Geni discussions integration?

### DISCUSSION THREAD

**Discussion by Sternbach-Software** [Sternbach-Software/GeniForAndroid#2] [General]
This is a place to discuss the necessity/convenience of supporting Geni discussions. Understand that this is not a trivial feature, and adding integration will increase the workload for maintaining the app, and will take away time from working on core features. I haven't looked if they have an API for this, but if there isn't one yet and there is enough demand, they may make one.

**Comment by carsrac** [comment]
As already indicated, add this feature would add add a lot of work at the Geni programming staff and on the app programming staff.
